packages/mask/src/plugins/RedPacket/SNSAdaptor/RedPacketNft.tsx

```diff
@@ -395,6 +398,12 @@ export function RedPacketNft({ payload }: RedPacketNftProps) {
                                     erc721TokenDetailed?.info.mediaUrl ? '' : classes.loadingTokenImg,
                                 )}
                                 src={erc721TokenDetailed?.info.mediaUrl ?? previewNftImg}
+                                onError={(e) => {
+                                    ;(e.target as any).src = new URL(
+                                        './assets/nft-preview.png',
+                                        import.meta.url,
+                                    ).toString()
+                                }}
```

Comment:
```suggestion
                                onError={(event) => {
                                    const target = event.target as HTMLImageElement
                                    target.src = new URL(
                                        './assets/nft-preview.png',
                                        import.meta.url,
                                    ).toString()
                                }}
```

Reply:
@zhouhanseng Use `currentTarget` instead, which referrers to current `img` element, and `typescript` will understand it has the `src` attribute.
```suggestion
                                onError={(e) => {
                                    e.currentTarget.src = new URL(
                                        './assets/nft-preview.png',
                                        import.meta.url,
                                    ).toString()
                                }}
```